onnx_tf/opr_checker.py

```diff
@@ -0,0 +1,157 @@
+from onnx_tf.pb_wrapper import TensorflowNode
+
+warnings.filterwarnings('ignore')
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger()
+
+
+def parse_args(args):
+  parser = argparse.ArgumentParser(
+      description=
+      'This is the checker to check whether model graph '\
+      'operators are supported by the frontend handlers'
+  )
+  parser.add_argument(
+      "--infile",
+      "-i",
+      help="Input file path, can be pb, pbtxt, or ckpt file.",
+      required=True)
+
+  return parser.parse_args(args)
+
+
+def check_opr_support(graph_def):
+  """ Check support for operators in graph
+
+  :param graph_def: the graph of operations
+  :return: whether all operators are supported, supported operators in graph
+  """
+
+  logger.info('Checking for unsupported operators...')
+  node_dict = set()
+  unsupported = set()
+  supported = set()
+  for node in graph_def.node:
+    node_dict.add(node.op)
+
+  node_dict.discard('Placeholder')
+  node_dict.discard('Const')
+
+  logger.info('There are %s unique operators in the model file.',
+              str(len(node_dict)))
+  for k in node_dict:
+    if k not in opset_version.frontend_tf_opset_version:
+      unsupported.add(k)
+    else:
+      supported.add(k)
+
+  if unsupported:
+    logger.info(
+        'There are %s operators currently not supported by the '\
+        'ONNX-Tensorflow frontend for your model.',
+        str(len(unsupported)))
+    logger.info(unsupported)
+    return False, supported
+  logger.info('All operators in the model are supported!')
+  return True, supported
+
+
+def check_node_args(graph_def, supported):
+  """ Check for required node arguments in graph
+
+  :param graph_def: the graph of operations
+  :param supported: the supported operators in graph
+  :return: whether all required parameters are provided
+  """
+
+  logger.info('Checking for required node arguments...')
+
+  opset_dict = {}
+  opset_dict[defs.ONNX_DOMAIN] = defs.onnx_opset_version()
+  handlers = get_all_frontend_handlers(opset_dict)
+
+  total_nodes = 0
+  failed_nodes = 0
+  for node in graph_def.node:
+    if node.op in supported:
+      total_nodes += 1
+      tf_node = TensorflowNode(node)
+      kwargs = {}
+      for inp in node.input:
+        for attr_node in graph_def.node:
+          if inp == attr_node.name:
+            kwargs[inp] = attr_node.attr['value']
+            break
+      handler = handlers.get(defs.ONNX_DOMAIN, {}).get(node.op, None)
+      try:
+        handler.args_check(tf_node, consts=kwargs)
+      except Exception as e:
+        logger.info(e)
+        failed_nodes += 1
+
+  logger.info('We checked %d supported nodes for required arguments.',
+              total_nodes)
+  logger.info('  # of nodes passed the args check: %d',
+              total_nodes - failed_nodes)
+  logger.info('  # of nodes failed the args check: %d', failed_nodes)
+  return failed_nodes == 0
+
+
+def load_graph_from_ckpt(ckpt_file):
```

Comment:
Good.
I will refactor later. `load_graph_from_ckpt` is also needed in `converter.convert`.